src/UBC Toboggan/Assets/Scripts/StopWatch.cs

```diff
@@ -21,10 +25,11 @@ void Update()
         if (isTimerRunning)
         {
             secondsElapsed += Time.deltaTime;
-            if (secondsElapsed > 90) // Consider setting the limit to 7.5 minutes (450 seconds)
+            if (secondsElapsed > 60) // Consider setting the limit to 7.5 minutes (450 seconds)
```

Comment:
Please remove the comment beside this if statement since we'll no longer be doing that